libraries/botbuilder-dialogs/tests/test_choice_prompt.py

```diff
@@ -267,11 +267,8 @@ def test_choice_prompt_with_none_id_should_fail(self):
         dialogs.add(choice_prompt)
 
         step1 = await adapter.send(Activity(type=ActivityTypes.message, text="Hello"))
-        # TODO ChoiceFactory.inline() is broken, where it only uses hard-coded English locale.
-        # commented out the CORRECT assertion below, until .inline() is fixed to use proper locale
-        # step2 = await step1.assert_reply('Please choose a color. (1) red, (2) green, o (3) blue')
         step2 = await step1.assert_reply(
-            "Please choose a color. (1) red, (2) green, or (3) blue"
+            "Please choose a color. (1) red, (2) green, o (3) blue"
```

Comment:
Is this 'o' expected behavior?

Reply:
Yes, 'o' is expected in this particular test case, because it's testing 'es-es' locale :)

Reply:
Although it admittedly does look odd that most of the sentence is in English, except the "inline or more separator"